components/applications-service/pkg/storage/postgres/service_internal_test.go

```diff
@@ -4,9 +4,39 @@ import (
 	"regexp"
 	"testing"
 
+	"github.com/chef/automate/components/applications-service/pkg/storage"
 	"github.com/stretchr/testify/assert"
 )
 
+func TestConvertComposedServiceToStorage(t *testing.T) {
+	expected := &storage.Service{}
+	assert.Equal(t, expected, convertComposedServiceToStorage(&composedService{}))
+
+	expected = &storage.Service{ID: 1, SupMemberID: "foo"}
+	assert.Equal(t, expected, convertComposedServiceToStorage(&composedService{ID: 1, SupMemberID: "foo"}))
+
+	expected = &storage.Service{Application: "abcd", PreviousHealth: "MORE-OR-LESS-OK"}
+	assert.Equal(t, expected, convertComposedServiceToStorage(&composedService{
+		Application: "abcd", PreviousHealth: "MORE-OR-LESS-OK", Status: "",
```

Comment:
There's little value in providing `Status: ""`, since that's what it is *if unset* anways. Unless there's a reason to be explicit here.

Reply:
Yes, mainly I am testing that when there are set values it converts them and when there is NO value, it also respects them. Thoughts?

Reply:
So, better be explicit :+1: that's ok.